config/_default/params.yaml

```diff
@@ -10,32 +10,49 @@ code_languages:
   'py':
     extension: py
     name: python
+    syntax: python
+  'pybeta':
+    extension: py
+    name: python-beta
+    syntax: python
   'rb':
     extension: rb
     name: ruby
+    syntax: ruby
+  'rbbeta':
+    extension: rbbeta
+    name: ruby-beta
+    syntax: ruby
   'sh':
     extension: sh
     name: curl
+    syntax: curl
   'go':
     extension: go
     name: go
+    syntax: go
   'java':
     extension: java
     name: java
+    syntax: java
   'cs':
     extension: cs
     name: csharp
+    syntax: csharp
   'php':
     extension: php
     name: php
+    syntax: php
 
 code_language_ids:
     java: Java
     cpp: 'C++'
     php: PHP
     nodejs: NodeJS
     python: Python
+    python-beta: Python [beta]
```

Comment:
@davidejones @apigirl we can modify the text for new Python and Ruby API clients here.